docs/howtos.md

```diff
@@ -0,0 +1,214 @@
+
+```
+openssl req -x509 -newkey rsa:2048 -keyout pilosa.local.key -out pilosa.local.crt -days 3650 -nodes -subj "/C=US/ST=Texas/L=Austin/O=Pilosa/OU=Com/CN=*.pilosa.local"
+```
+
+The command above creates two files in the current directory:
+* `pilosa.local.crt` is the SSL certificate.
+* `pilosa.local.key` is the private key file which must be kept as secret.
+
+Having created the SSL certificate, we can now create the gossip encryption key. Gossip encryption key file must be exactly 16, 24, or 32 bytes to select one of AES-128, AES-192, or AES-256 encryption. Reading random bytes from cryptographically secure `/dev/random` serves our purpose very well:
+```
+head -c 32 /dev/random > pilosa.local.gossip32
+```
+
+We now should have `pilosa.local.gossip32` in the current directory with 32 random bytes.
+
+#### Creating the Configuration Files
+
+Pilosa supports passing configuration items using the command line, environment variables or a configuration file. We will use the last option in this tutorial and create three configuration files for our three nodes.
+
+Create `node1.config.toml` in the project directory and paste the following in it:
+
+```toml
+# node1.config.toml
+
+data-dir = "node1_data"
+bind = "https://01.pilosa.local:10501"
+
+[cluster]
+hosts = ["https://01.pilosa.local:10501", "https://02.pilosa.local:10502", "https://03.pilosa.local:10503"]
+
+[tls]
+certificate = "pilosa.local.crt"
+key = "pilosa.local.key"
+skip-verify = true
+
+[gossip]
+seed = "01.pilosa.local:15000"
+port = 15000
+key = "pilosa.local.gossip32"
+```
+
+Create `node2.config.toml` in the project directory and paste the following in it:
+
+```toml
+# node2.config.toml
+
+data-dir = "node2_data"
+bind = "https://02.pilosa.local:10502"
+
+[cluster]
+hosts = ["https://01.pilosa.local:10501", "https://02.pilosa.local:10502", "https://03.pilosa.local:10503"]
+
+[tls]
+certificate = "pilosa.local.crt"
+key = "pilosa.local.key"
+skip-verify = true
+
+[gossip]
+seed = "01.pilosa.local:15000"
+port = 16000
+key = "pilosa.local.gossip32"
+```
+
+Create `node3.config.toml` in the project directory and paste the following in it:
+
+```toml
+# node3.config.toml
+
+data-dir = "node3_data"
+bind = "https://03.pilosa.local:10503"
+
+[cluster]
+hosts = ["https://01.pilosa.local:10501", "https://02.pilosa.local:10502", "https://03.pilosa.local:10503"]
+
+[tls]
+certificate = "pilosa.local.crt"
+key = "pilosa.local.key"
+skip-verify = true
+
+[gossip]
+seed = "01.pilosa.local:15000"
+port = 17000
+key = "pilosa.local.gossip32"
+```
+
+Here is some explanation of the configuration items:
+* `data-dir` points to the directory where the Pilosa server writes its data. If it doesn't exist, the server will create it.
+* `bind` is the address which the server listenes to for incoming requests. The address is composed of three parts, the scheme, host, and port. The default scheme is `http` so we explicitly specify `https` to use the HTTPS protocol for communication between nodes.
```

Comment:
i would suggest (also note spelling correction in `listenes`.
```
`bind` is the address to which the server listens for incoming requests. The address is composed of three parts: scheme, host, and port. The default scheme is `http` so we explicitly specify `https` to use the HTTPS protocol for communication between nodes.
```

Reply:
@travisturner Thanks, updated.